migration/20180105-2-materialized-work-for-lanes.sql

```diff
@@ -1,10 +1,4 @@
--- The default for lanes.inherit_parent_restrictions has changed
--- to true. Change the setting for all lanes not based on custom lists.
--- For all lanes based on custom lists, a human has made a decision
--- to set the value one way or the other.
-update lanes set inherit_parent_restrictions=true where _list_datasource_id is null and id not in (select lane_id from lanes_customlists);
-ALTER TABLE lanes ALTER COLUMN inherit_parent_restrictions SET DEFAULT true;
-
```

Comment:
I moved this into a separate file so that the rest of this file could be a copy of `materialized_view_for_lanes.sql`.

Reply:
You're missing the file for this.